index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap"
+      rel="stylesheet"
+    />
+    <title>Stanley Personal Portfolio</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <nav class="header_bar heading heading-pl">
+        <a href="" class="logo_info">Stanley</a>
+        <button class="logo_vector-btn">
+          <img src="images/Icon - Menu.png" alt="Vector" />
+        </button>
+      </nav>
+    </header>
```

Comment:
- [ ] You did a great job as per semantics in this section! Kudos for that! 💯 However, there are still some issues with the design as shown below:

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/205151857-afca266f-9969-4d61-81f8-6d2c5fd6d8e2.png) | ![image](https://user-images.githubusercontent.com/77069011/205151705-eb6bf640-838e-4c92-8d52-cd06960af189.png) |

Please consider using the same font size, color, and weight as the design, and also your hamburger button needs some styling, kindly remove the border. One of the main goals of this first project is that you demonstrate the ability to follow a given design as closely as possible, so please consider making these corrections! 🤓 🛠️